go.mod

```diff
@@ -156,3 +158,7 @@ replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 replace github.com/cosmos/cosmos-sdk/db => ./db
 
 retract v0.43.0
+
+replace github.com/tendermint/tendermint => github.com/yihuang/tendermint v0.35.0-rc1.0.20220311084540-53e8a01f4650
```

Comment:
TODO: wait for upstream PR to be merged and backported: https://github.com/tendermint/tendermint/pull/8101